docs/evaluate/development-production-features/workflow-message-passing.mdx

```diff
@@ -15,34 +16,41 @@ keywords:
   - business logic activities
   - temporal workers
   - temporal SDK tutorial
-  - go sdk temporal guide
-  - java sdk temporal guide
-  - python sdk temporal guide
-  - typescript sdk temporal guide
   - learning temporal workflows
   - developing with temporal
   - temporal workflow execution
   - temporal activity management
   - worker process execution
   - signals
   - queries
+  - updates
 ---
 
 import { RelatedReadContainer, RelatedReadItem } from '@site/src/components/related-read/RelatedRead';
 
-Need a responsive Workflow?
-Temporal provides **Signals**, **Queries**, and **Updates** as a means to send data into and get data out of your running Workflow.
-Signals enable your Workflow to asynchronously react to events at runtime, Queries enable on-demand data retrieval from ongoing Workflows without disrupting their execution, and Updates enable you to change the state of a running Workflow.
+Need a responsive Workflow? Think about use cases like these:
```

Comment:
I don't quite get what the first question is saying, but it may just be that I'm slightly resistant to marketing-style copy. Is it like

> Sometimes you need to be able to interact with your running Workflows. Think about use cases like these: